controllers/userController.js

```diff
@@ -62,13 +63,15 @@ exports.login = async (req, res, next) => {
 exports.changePassword = async (req, res, next) => {
   try {
     const { resetPasswordToken, newPassword } = req.body;
-    const user = await userService.findUser({resetPasswordToken});
+    const user = await userService.findUser({resetPasswordToken: resetPasswordToken});
+    console.log(user)
     if (!user) {
       return res.status(400).json({
         message: "Invalid or expired reset token",
       });
     }
     await userService.updatePassword(user, newPassword);
```

Comment:
When creating services for specific models, it's advisable to design queries in a versatile manner that can serve multiple purposes. For instance, if you're developing a service for updating the user model and using a query like findOneAndUpdate, it's beneficial to make this query more general and give it a generic name. This approach allows you to utilize the same query for various purposes related to updating the user model, eliminating the need for multiple distinct services